internal/cmd/operator-sdk/bundle/validate/cmd.go

```diff
@@ -63,7 +68,14 @@ To list and run optional validators, which are specified by a label selector:
   NAME           LABELS                     DESCRIPTION
   operatorhub    name=operatorhub           OperatorHub.io metadata validation
                  suite=operatorframework
+
+To validate a bundle with a superset of requirements for Operator Framework: 	
+	
   $ operator-sdk bundle validate ./bundle --select-optional suite=operatorframework
+
+To validate a bundle with a superset of requirements for *OperatorHub.io* specifically:
+	
+  $ operator-sdk bundle validate ./bundle --select-optional suite=operatorframework,name=operatorhub
```

Comment:
The `suite` argument is redundant here because `name` is more specific.
```suggestion
  $ operator-sdk bundle validate ./bundle --select-optional name=operatorhub
```